src/fmu/dataio/_definitions.py

```diff
@@ -59,12 +59,14 @@ def __post_init__(self):
     "volumetrics": None,  # or?
     "khproduct": None,
     "timeseries": None,
+    "wellpicks": None,
 }
 
 STANDARD_TABLE_INDEX_COLUMNS = {
     "inplace_volumes": ["ZONE", "REGION", "FACIES", "LICENCE"],
     "timeseries": ["DATE"],  # summary
     "rft": ["measured_depth", "well", "time"],
+    "wellpicks": ["WELL", "HORIZON"],
```

Comment:
I would think that MD is (more or less) required? (I have seen well picks with X Y Z in rare cases, but then the software need to compute MD)

Reply:
Yes, correct. Added the `MD` column to the test data. However, the `STANDARD_TABLE_INDEX_COLUMNS` is referring to (categorical) index columns, which would still be just "WELL" and "HORIZON" in this case. So line `69` should not change (?)